rta/__init__.py

```diff
@@ -39,6 +45,116 @@ def get_ttp_names(os_types=None):
     return names
 
 
+def get_available_tests(print_list=False, os_filter=None):
+    test_names = []
+    test_metadata = []
+
+    for file in list(CURRENT_DIR_PATH.rglob("*.py")):
+        if file.stem not in ("common", "main") and not file.stem.startswith("_"):
+            module = importlib.import_module(f"rta.{file.stem}")
+            if os_filter and os_filter not in module.PLATFORMS:
+                continue
+            test_names.append(file.stem)
+            test_metadata.append({"name": file.stem, "path": file, "platforms": module.PLATFORMS})
+
+    if print_list:
+        longest_test_name = len(max(test_names, key=len))
+        print("Printing available tests")
+        header = f"{'name':{longest_test_name}} | {'platforms':<30}"
+        print(header)
+        print("=" * len(header))
+        [print(f"{test['name']:<{longest_test_name}} | {', '.join(test['platforms'])}") for test in test_metadata]
+
+    return test_names, test_metadata
+
+
+def get_rta_triggered_rules() -> dict:
+    # get all rtas and the rules they cover
+    all_rtas = {"windows": [], "linux": [], "macos": []}
+    for file in list(CURRENT_DIR_PATH.rglob("*.py")):
+        if file.stem not in ("common", "main") and not file.stem.startswith("_"):
+            module = importlib.import_module(f"rta.{file.stem}")
+            for platform in module.PLATFORMS:
+                all_rtas[platform].extend(module.TRIGGERED_RULES.get("SIEM"))
+    return all_rtas
+
+
+def get_os_list(rule) -> list:
```

Comment:
This can't work without the detection_rules code. Is it necessary though? Why not just parse the rule_id and os type from the RTA, since you are adding it anyway